### ISSUE TITLE
Item disappearing

### ISSUE THREAD

**Issue by Danpeal** [raoulvdberge/refinedstorage#1285]
#### Issue description:
Items are disappearing when entered manually in the Grid

#### What happens:
I crafted some items in the RS Grid and dropped them in to RS via Grid. After that I wanted to use those items to do the next recepie and I could not find those items anymore.
Tried is with other stuff like sugarcan -> 140 in the system, took 32 out and put them in again -> vanished only 108 found in the system.

#### What you expected to happen:
At least I should have be able to see all of the Items i produced or sent back to system
#### Steps to reproduce:
1. Take stuff out of RS
2. Put stuff back in RS
3. Stuff vanishes
...
Additional information:
I checked all storage cubes and they are all on In / out
Have some Vast storages and also there all Storage are on In/ out
Have 1 storage drive additionally and also there In / out active
Filters are set in the cubes to accept only 1 item.
Vast Storage are also set to 1 item
Storage driver should accept everything else.

#### Version (make sure you are on the latest version before reporting):

- Minecraft: 1.10.2
- Forge: 12.18.3.2281
- Refined Storage: 1.2.25
- Foolcraft Pack: 1.3.3

Does this issue occur on a server? yes

#### If a (crash)log is relevant for this issue, link it here:

[pastebin/gist/etc link here]

**Comment by raoulvdberge** [comment]
I can't reproduce issues as these.

**Comment by Tkdmaster** [comment]
I'd like to report this problem exists for me as well. I have a RS system setup, and it seems to be continually eating random item stacks. I am using the Galactic Engineers 1.10.2 pack and crafted a lot of stuff for quests early on, all of that stuff has disappeared now. I recently made a stack of advanced Rocketry engines and I went to get them, all gone overnight.

I cannot specify any re-producable steps to this beyond adding several 64k drives to a system, and filling them with a ton of different types of items and stacks of them. 

This is a quite serious issue and really should be looked into.
